stratum/lib/utils.cc

```diff
@@ -20,6 +20,7 @@
 #include "absl/strings/str_split.h"
 #include "stratum/lib/macros.h"
 #include "stratum/public/lib/error.h"
+#include "p4/v1/p4runtime.pb.h"
```

Comment:
Do you see any benefit in moving this to the hal/lib/p4/utils.h file?

Reply:
Make sense